src/index.js

```diff
@@ -80,6 +82,41 @@ const edit = () => {
     });
   });
 };
+
+// Check Completed value
+const markCompleted = () => {
+  let completedValue = false;
+  const checkInputs = document.querySelectorAll('.checkInput');
+  checkInputs.forEach((checkbox) => {
+    checkbox.addEventListener('click', () => {
+      /* eslint-disable no-unused-expressions */
+      checkbox.checked ? completedValue = true : completedValue = false;
+      const key = checkbox.parentElement.parentElement.getAttribute('key');
+      const result = tasks.find((task) => {
+        return String(key) === String(task.index);
+      });
+      // result.description = result.description;
+      result.completed = completedValue;
+      const convertedTasks = JSON.stringify(tasks);
+      localStorage.setItem('data', convertedTasks);
+      window.location.reload();
+    });
+  });
+};
+
+// Clear All Completed Tasks
+clearBtn.addEventListener('click', () => {
+  const uncompletedTasks = tasks.filter((item) => item.completed === false);
+
+  tasks = uncompletedTasks;
+  if (uncompletedTasks.length) {
+    localStorage.setItem('data', JSON.stringify(uncompletedTasks));
+  } else {
+    localStorage.clear();
+  }
+  window.location.reload();
+});
+
```

Comment:
- Kindly create a new JavaScript file and import it as a module. The new functionalities should be in the new file as per project requirements.

![image](https://user-images.githubusercontent.com/38088818/193400540-f2d5c230-33d2-4a79-a8b5-dfab96542884.png)